spec/models/sitemap_spec.rb

```diff
@@ -0,0 +1,72 @@
+require 'spec_helper'
+
+describe Sitemap do
+  let!(:url) { 'http://www.agency.gov/boring' }
```

Comment:
P.S. I'm guessing you snagged `http://www.agency.gov/boring` from one of my specs. I was apparently feeling snarky that day; let's use something more, ah, boring, such as: `http://www.agency.gov/doc1`.